test/test-applications/calltarget/CallTargetNativeTest/CallTargetNativeTest.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+	  <LangVersion>latest</LangVersion>
```

Comment:
nit: Looks like a tab snuck in here 😛

Reply:
:) thanks

Reply:
fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1007/commits/04ae61a340872f36f5e93929e69b26305bc991dd